joomla/joomla-cms
# [com_newsfeeds] Add search tooltip

#### Summary of Changes

Simple PR to add the search tooltip and element label (accessibility) in com_newsfeeds.
#### Testing Instructions

Code review, or:
1. Apply patch
2. Go to Components -> News Feeds -> Feeds, check the tooltip in the search field.
